mypy/report.py

```diff
@@ -35,7 +35,7 @@
     import lxml.etree  # type: ignore
     from lxml import etree  # type: ignore
     LXML_INSTALLED = True
-except ImportError:
+except Exception:  # ImportError, or SystemError (py38-dev currently)
```

Comment:
Not really sure about this one.. could/should at least be `(ImportError, SystemError)` then maybe.
Fixes https://travis-ci.org/python/mypy/jobs/540201447#L358

Reply:
A system error implies something is very broken (often an internal interpreter error). We should not be catching SystemError. I think this implies we should just not install lxml under 3.8 until it is fixed. I'd recommend modifying `test-requirements.txt` so that it looks like `lxml; python_version < '3.8'`. Then lxml will not be installed and we won't have to deal with it being broken on 3.8.